Cargo.toml

```diff
@@ -37,6 +39,7 @@ hash-db = "0.16"
 hex = "0.4"
 ic-agent = "0.24"
 ic-canister = { git = "https://github.com/infinity-swap/canister-sdk", package = "ic-canister", tag = "v0.3.x" }
+ic-storage = { git = "https://github.com/infinity-swap/canister-sdk", package = "ic-storage", tag = "v0.3.x" }
```

Comment:
Could you please replace `infinity-swap` with `bitfinity-network` in all toml files?

Reply:
Done